torch/autograd/__init__.py

```diff
@@ -26,32 +26,50 @@
 __all__ = ['Variable', 'Function', 'backward', 'grad_mode']
 
 _OptionalTensor = Optional[torch.Tensor]
+_ShapeorNestedShape = Union[_size, Sequence[_size]]
+
+
+def _calculate_shape(output: torch.Tensor, grad: torch.Tensor,
```

Comment:
We won't be able to test the nested path without the next PR right? Either move all changes in __init__.py there, or split it out so that non-nested changes goes in the refactor-only PR and only nested-specific changes (the nested path) go in the last PR.

Reply:
Will remove changes from this PR